opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -117,8 +117,7 @@ class EclMaterialLawManager
                       const std::vector<int>& compressedToCartesianElemIdx)
     {
         // get the number of saturation regions and the number of cells in the deck
-        int ntsFun = deck.getKeyword("TABDIMS").getRecord(0).getItem("NTSFUN").get<int>(0);
-        unsigned numSatRegions = static_cast<unsigned>(ntsFun);
+        const size_t numSatRegions = eclState.runspec().tabdims().getNumSatTables();
```

Comment:
> Just generally trying to move towards a less noisy interface.

I also generally prefer to skip the `get` prefix. That said, IMO it is more important to keep the API conventions somewhat consistent and also I have only a limited desire to do something like "a great `get` purge", so I vote for adding the `get` prefix to the methods that lack it (e.g. `runspec()` -> `getRunspec()`)...

> The get-prefix'd still exist mostly for compatibility at this point (in my not so humble opinion)

IMO, if you care enough to put effort into this, non-`get` variants can be added to everything and the `get` variants of the methods could be kept around for a while but marked deprectated (so that people will get warnings instead of errors).